### PR TITLE
Troubleshooting: checking whether job is admitted or preempted

### PR DESCRIPTION
**Pull request by alculquicondor**
#### What type of PR is this?

/kind documentation

#### Which issue(s) this PR fixes:

Part of #1410

#### Special notes for your reviewer:

This is very basic for now, but we have to start somewhere.

I plan to follow up with more details for misconfigured ClusterQueues and another page dedicated to Pods, as they are very different from the rest of the jobs.

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: site/content/en/docs/tasks/_index.md [hunk 1]
```diff
@@ -50,3 +50,10 @@ A _platform developer_ integrates Kueue with other software and/or contributes t
 As a platform developer, you can learn how to:
 - [Integrate a custom Job with Kueue](/docs/tasks/integrate_a_custom_job).
 - [Enable pprof endpoints](/docs/tasks/enabling_pprof_endpoints).
+
+## Troubleshooting
+
+Sometimes things go wrong. The following guides can help you understand the state
+of the system.
+
+- [Troubleshooting Jobs](troubleshooting/troubleshooting_jobs)
```
Comment (PBundyra) [inline site/content/en/docs/tasks/_index.md:59]: On the deploy preview this link does not work correctly

Comment (alculquicondor) [reply site/content/en/docs/tasks/_index.md:59]: I'm not sure why it's not showing yet. I'm checking

Comment (alculquicondor) [reply site/content/en/docs/tasks/_index.md:59]: ahhh, it was because I was using a future date :)

**Comment by PBundyra** [comment]
LGTM

**Pull request merged**